### DISCUSSION TITLE
SSL_CTX_new() failed with producer with openssl 1.1.1

### DISCUSSION THREAD

**Discussion by write2jaydeep** [edenhill/librdkafka#3777] [General]
We have upgraded openssl 1.1.1L  from openssl 1.1.0. now we are unable to start producer.

We have below errors:
**1647970914.706 ARI-RDKAFKA-3-SSL: rdkafka#producer-1: [thrd:app]: ssl_ciph.c:1516: error:140A6041:lib(20):func(166):reason(65)
% Failed to create new producer: SSL_CTX_new() failed: ssl_lib.c:1983: error:140A90A1:lib(20):func(169):reason(161)**

Linux System
version: 1.9.0

Can someone help me?

**Comment by edenhill** [comment]
```bash
$ openssl errstr 140A90A1 140A6041
error:140A90A1:SSL routines:SSL_CTX_new:library has no ciphers
error:140A6041:SSL routines:ssl_create_cipher_list:malloc failure
```

**Reply by write2jaydeep** [reply]
Thank you very much! I had a conflict between the library and the program. The library is built with openssl 1.1.1 and my program was built with openssl1.0.0. 
Now it has been resolved.
